breakdancer.js

```diff
@@ -94,4 +94,54 @@ export default class Breakdancer {
 
     return 'unknown';
   }
+
+  /**
+   * Compare current and specified properties of given breakpoint
+   * Requires that this.height() and this.width() are implemented
```

Comment:
wonder if we should safeguard against this, although both native and regular have default implementations.

Reply:
I was thinking that because this is already a `@private` method, we can safeguard this by relying on the fact `this.height()` and `this.width()` methods **already exist** in `index.js` and `index.native.js`.

However, it would be worthwhile to throw a `ReferenceError` if the associated `this[property]` function did not exist.